src/views/tickets/rsvp.php

```diff
@@ -6,12 +6,13 @@
  *
  *     [your-theme]/tribe-events/tickets/rsvp.php
  *
+ * @since 4.0
  * @since 4.10.8 More similar display format to that of other ticket types, including better checking of max quantity available.
  *
  * @version 4.10.8
```

Comment:
This view needs a `@since TBD __what changed__` and `@version TBD`

Reply:
done